gnucash/import-export/test/gtest-import-backend.cpp

```diff
@@ -0,0 +1,273 @@
+#include <gtk/gtk.h>
+#include <gtest/gtest.h>
+#include <gmock/gmock.h>
+
+#include <config.h>
+
+#include <gnc-datetime.hpp>
+
+extern "C"
+{
+#include <import-backend.h>
+#include <engine-helpers.h>
+#include <gnc-ui-util.h>
+}
+
+#include "gmock-gnc-prefs.h"
+#include "gmock-qofbook.h"
+#include "gmock-Account.h"
+#include "gmock-Transaction.h"
+#include "gmock-Split.h"
+
+
+
+/* Global test environment */
+
+class TestEnvironment : public testing::Environment
+{
+public:
+    void SetUp()
+    {
+        m_book = new QofMockBook;
+    };
+
+    void TearDown()
+    {
+        m_book->free();
+    };
+
+    QofMockBook* m_book;
+};
+
+testing::Environment* const env = testing::AddGlobalTestEnvironment(new TestEnvironment);
+
+
+
+/* mock functions, which can not be mocked by mock classes */
```

Comment:
They're fakes/replacements, not mocks, and the comment should say why they're needed, e.g. being defined in a library we don't want to link to.

Reply:
Correct

Reply:
The point is that the comment needs to say that.

Reply:
Changed in PR #738